localization/english/CWP/africa/CWP_libya_l_english.yml

```
@@ -0,0 +1,140 @@
+  libya.7.f: "The times of Arab nationalism and unity are gone forever. These ideas which mobilized the masses are only a worthless currency. Libya has had to put up with too much from the Arabs for whom it has poured forth both blood and money.\n― Muammar Gaddafi"
+  libya.7.a: "Traitors!"
+
+  libya.8.t: "[SCOPE.sCountry('gaddafi_unification_target_scope').GetName] rejects our Offer"
+  libya.8.d: "In a tense turn of events, Muammar Gaddafi contemplates a daring coup attempt against the government of [SCOPE.sCountry('gaddafi_unification_target_scope').GetName], following their resounding refusal to join his proposed union. The rejection has sent shockwaves through diplomatic circles, exacerbating already strained relations between the neighboring nations."
+  libya.8.f: "I staged a coup d’état and the workers staged the revolution, occupying the factories, becoming partners instead of employees, eliminating the monarchic administration and setting up people’s committees. In short, they liberated themselves. Students did the same. Today in Libya, only the people count.\n― Muammar Gaddafi"
+  libya.8.a: "Attempt to overthrow their government"
+  libya.8.b: "We want nothing to do with traitors"
+
+  libya.9.t: "Gaddafis Influence spreads"
+  libya.9.d: "In a monumental display of unity, thousands of Libyans march peacefully towards our capital, driven by an unwavering determination to make their voices heard. Their resolute purpose? To compel our nation to join Gaddafi's union. The sheer magnitude of their collective willpower leaves no doubt that their mission will succeed."
+  libya.9.f: "Revolution is when the masses make the revolution. A people’s revolution. But even when the revolution is made by others in the name of the masses, expressing what the masses want, it can be revolution. A people’s revolution because it has the support of the masses and interprets the will of the masses.\n― Muammar Gaddafi"
+  libya.9.a: "We have no choice"gaddafi_unification_current_target
+
+  libya.100.t: "Diplomatic Pressure"
+  libya.100.d: "Our unification efforts are progressing slowly, and Gaddafi is growing impatient. With a little diplomatic pressure, we could gain more influence, but good relations are essential to the unification process."
+  libya.100.f: ""
+  libya.100.a: "Force them to cooperate"
+  libya.100.b: "Good relationships are more important"
+
+  libya.101.t: "The Green Revolution"
+  libya.101.d: "Progress at the negotiating table is slow, but there are reports of pro-Libyan rebels in the areas of our negotiating partner. We might be able to help them, but that would throw the country into chaos."
```

- Grammar: prefixed propper nouns should be capitalized, fixed passive voice in first scentence.
- Clarification Needed: In the second scentence who does "them" refer to. The rebels or the negotiating partner. Who's country would be thrown into chaos, ours, or our negotiating partners?
```suggestion
  libya.101.d: "While progress at the negotiating table has slowed, Pro-Libyan rebels have been reported in the areas controlled by our negotiating partner. We might be able to help them, but that would throw the country into chaos."
```